packages/compass-aggregations/src/stores/store.ts

```diff
@@ -176,15 +175,6 @@ export function activateAggregationsPlugin(
     )
   );
 
-  store.dispatch(
-    preferencesMaxTimeMSChanged(preferences.getPreferences().maxTimeMS)
-  );
```

Comment:
Do we still apply the initial maxTimeMS value when the store was created somewhere?

Reply:
Otherwise, I think the `maxTimeMS` value from the preferences might not end up being applied, e.g. for the explain call in modules/insights.ts?

Reply:
added [3ac15a9](https://github.com/mongodb-js/compass/pull/5551/commits/3ac15a9898f1244d9848f951cfa04d968b816f97) to apply initial maxTimeMS from preferences (or the default one).